src/Externals/Numeral.res

```diff
@@ -169,67 +169,67 @@ module String = Make({
   type input = string
   type output = Js.Nullable.t<float>
   type parsedOutput = option<float>
-  let parseOutput = Js.Nullable.toOption
+  let parseOutput = output => Js.Nullable.toOption(output)
 })
 /* It probably doesn't make sense to add other types, like `int`, because the
  we can't guarantee that it will stay an int once it goes to the JS side. */
-module Helpers = {
-  /* These bindings aren't well tested or documented. Use with caution.
-   Pull requests and bug reports are welcome! */
-  type t
-  @get external getHelpers: numeral => t = "_"
+// module Helpers = {
```

Comment:
Couldn't get these to compile at all, no idea why

Reply:
Ugh. This code was always pretty messy and I don't recall the details of how/why I designed it now. I can try to take a look at it later when I get a chance.